apps/governance/src/routes/staking/home/validator-tables/standby-pending-validators-table.tsx

```diff
@@ -248,11 +247,13 @@ export const StandbyPendingValidatorsTable = ({
     );
 
     return (
-      <div data-testid="standby-pending-validators-table">
+      <div
+        data-testid="standby-pending-validators-table"
+        className="validators-table"
```

Comment:
non-blocking: why we need this className? Can't see anything related to it in this PR. Maybe some styles from `customThemeParams`, are being removed here, should be implemented using it?

Reply:
Well spotted this was for some custom styles for some aggrids on the governance app. This has been merged so I will make a follow up Pr